docker-compose.yaml

```
@@ -0,0 +1,74 @@
+    networks:
+      - frontnet
+    # volumes:
+    #   - ./ssl:/etc/nginx/ssl:ro
+    #   - ./nginx-logs:/var/log/nginx
+
+volumes:
+  db-data:
+
+networks:
+  backnet:
+  frontnet:
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)